packages/extensionmanager/src/widget.tsx

```diff
@@ -19,6 +19,7 @@ import ReactPaginate from 'react-paginate';
 
 import { ListModel, IEntry, Action } from './model';
 import { isJupyterOrg } from './query';
+import { iconStyle } from '@jupyterlab/ui-components/lib/style/icon';
```

Comment:
We should never import from `lib`. Something like the above sometimes happens to me when the typescript autocomplete fills out the import, and I have to change it back to something like`import { iconStyle } from '@jupyterlab/ui-components';`

Reply:
Fixed! Turns out I didn't have the exports from the `src/style` subpkg set up correctly in the various `index.ts` files, which is why the autocomplete went for the funky `.../lib/...` import in the first place